server/plugins/stencil-editor/stencil-editor.module.js

```diff
@@ -30,6 +31,7 @@ module.exports.register = function (server, options, next) {
     options.themeServer.ext('onRequest', handlers.onRequest);
 
     // When using stencil-cli variationId = configurationId
+    configurationId = themeConfig.variationIndex + 2;
```

Comment:
the configId isn't going to be used at all, so I just made it some random thing. I can rename it to be `config`, like how you have `theme` as the versionId